hack/scale_out_poc/nginx.conf

```diff
@@ -0,0 +1,104 @@
+user www-data;
```

Comment:
I think we agreed to dump nginx and use haproxy. So I guess remove this file and everything about nginx when migrating changes to master.

Reply:
Is there a work-item or issue tracking this task?

Reply:
deleted